spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin2/ZipkinProperties.java

```diff
@@ -37,6 +37,13 @@
 	 */
 	private String baseUrl = "http://localhost:9411/";
 
+	/**
+	 * Api path that will be appended to baseUrl above as suffix This is applicable if you
```

Comment:
Change to:

The API path to append to the baseUrl (above) as a suffix. This applies if you